orga/tests/unit/routes/application-test.js

```diff
@@ -45,6 +46,14 @@ module('Unit | Route | application', function (hooks) {
       assert.ok(this.route.featureToggles.load.called);
     });
 
+    test('loads identity providers', async function (assert) {
+      // given // when
```

Comment:
```suggestion
      // when
```

cf. les Standards de l’équipe : 
> S’il n’y a pas de préparation nécessaire au test, c’est OK de ne pas mettre “given” (ne pas mettre “given & when” par exemple)